pkg/cluster/cluster.go

```diff
@@ -8,13 +8,14 @@ type ConnectArguments struct {
 	OfflineAccessToken      string
 	ForceCreationWithoutAsk bool
 	IgnoreContext           bool
-	SelectedKafka           string
 	Namespace               string
+	SelectedService         string
```

Comment:
[Responsibility]
Selected service is not argument/input (it is actually part of the logic). If someone uses this as lib it is not clear which one should be used (ID) and if SelectedService will be taken into account

Reply:
Can you elaborate this a bit?

Reply:
Closing this discussion as separation entails additional function argument.